lib/core/network/protocols/http.js

```diff
@@ -78,7 +85,33 @@ class HttpMessage {
     }
   }
 
+  /**
+   * Checks that an incoming HTTP message is well-formed
+   *
+   */
+  validate () {
+    if (this.isEmpty()) {
+      return;
+    }
+
+    const contentType = this.headers['content-type'];
```

Comment:
Should we consider the absence of `content-type` as an error?

Reply:
no, we default it to `application/json`